scripts/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# This is a generic build script for the GDK for Unity.
+# The expected usage is as follows:
+#   bash build.sh <unity_project_dir> <worker_type> <build_target> <log_file (optional)>
+
+set -e -u -x -o pipefail
+
+cd "$(dirname "$0")/../"
+
+source "scripts/pinned-tools.sh"
+source "scripts/profiling.sh"
+
+UNITY_PROJECT_DIR=$1
```

Comment:
This might be relative to the working directory when this script was invoked.

By adding `cd "$(dirname "$0")/../"` back in - this will break

Reply:
Not having cd... did break RUN_UNITY_PATH - dotnet complained about missing project.
Let me play with script to see what I can do to support relative path as well

Reply:
Sure - we probably want it to handle relative paths properly. It should be possible given:

1. We can get the path of this script
2. We can get the working directory it was invoked in

Reply:
Ok, this current version is verified to work on both relative and absolute paths. @jamiebrynes7 what do you think of it?

Reply:
Yeah that looks good to me :)